composer.json

```diff
@@ -131,7 +131,8 @@
     "bower-asset/select2-4.0.x": "~4.0.2",
     "bower-asset/slimScroll-1.3.x": "~1.3.7",
     "bower-asset/tinymce-builded": "^4.4",
-    "zeebinz/rarinfo": "~2.5.0"
+    "zeebinz/rarinfo": "~2.5.0",
+    "guzzlehttp/guzzle": "^6.3"
```

Comment:
FYI, if i add that manually, i need to run composer update, which i don't want to do, because it will update rest of the libs. 

So i used

 `composer require guzzlehttp/guzzle. `

Next time you plan updating composer libraries, just move that line to be sorted alphabetically.